inference/python/infbench/cutlass.py

```diff
@@ -96,30 +195,34 @@ def __init__(self, dataDir):
         self.M = 10000
         self.N = 8000
         self.K = 10000
+        self.A = [0 for i in range(self.ndata)]
+        self.B = [0 for i in range(self.ndata)]
 
     @property
```

Comment:
the @property thing is a bit of a hack (or at least some ugly and confusing syntax). You can just set "ndata = 20" in your class body.

What's happening is that Model is an abstract base class, but python is not very good at typing (it wasn't meant to be typed) so you have to resort to this weird "property" thing to convince the interpreter to check for required member variables.